bin/dev_stop.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo "Stopping Argo"
+if [ -e tmp/pids/server.pid ]; then
+    kill -9 $(cat tmp/pids/server.pid)
+    rm tmp/pids/server.pid
+    echo "Stopped Argo server"
+fi
+if [ -e tmp/pids/delayed_job.pid ]; then
+    kill -9 $(cat tmp/pids/delayed_job.pid)
+    rm tmp/pids/delayed_job.pid
+    echo "Stopped delayed_job"
+fi
```

Comment:
i'm still in favor of making these `kill`s instead of `kill -9`s, and printing the actual process ID from `server.pid` and `delayed_job.pid`.